mofacts/client/views/experiment/card.js

```diff
@@ -371,6 +371,17 @@ Template.card.rendered = async function() {
     Session.set('stimDisplayTypeMap', stimDisplayTypeMap);
   }
 
+  // Check if TDF allows for dialogue feedback preferences, if so, route to dialogue feedback widget
+  const allowFeedbackTypeSelect = Session.get('allowFeedbackTypeSelect');
+  const feedbackParamsSet = Session.get('feedbackParamsSet');
+  //Paused Lock to prevent audio from playing during preference selection
+  Session.set('pausedLocks', Session.get('pausedLocks')+1);
+  if(allowFeedbackTypeSelect && !feedbackParamsSet){
+    Router.go('/feedback');
+  } else {
+    Session.set('pausedLocks', Session.get('pausedLocks')-1);
```

Comment:
This will definitely have unintended consequences as we loop between card and instructions.html every unit. Go ahead and remove it